Wrappers/Python/cil/framework/label.py

```diff
@@ -0,0 +1,128 @@
+
+
+class ImageLabels(TypedDict):
+    RANDOM: str
+    RANDOM_INT: str
+    CHANNEL: str
+    VERTICAL: str
+    HORIZONTAL_X: str
+    HORIZONTAL_Y: str
+
+
+class AcquisitionLabels(TypedDict):
+    RANDOM: str
+    RANDOM_INT: str
+    ANGLE_UNIT: str
+    DEGREE: str
+    RADIAN: str
+    CHANNEL: str
+    ANGLE: str
+    VERTICAL: str
+    HORIZONTAL: str
+    PARALLEL: str
+    CONE: str
+    DIM2: str
+    DIM3: str
+
+
+class DataOrder(TypedDict):
+    ENGINES: List[str]
+    ASTRA_IG_LABELS: List[str]
+    TIGRE_IG_LABELS: List[str]
+    ASTRA_AG_LABELS: List[str]
+    TIGRE_AG_LABELS: List[str]
+    CIL_IG_LABELS: List[str]
+    CIL_AG_LABELS: List[str]
+    TOMOPHANTOM_IG_LABELS: List[str]
+
+    @staticmethod  # type: ignore[misc]
+    def get_order_for_engine(engine, geometry):
+        if engine == 'astra':
+            if isinstance(geometry, BaseAcquisitionGeometry):
+                dim_order = data_order["ASTRA_AG_LABELS"]
+            else:
+                dim_order = data_order["ASTRA_IG_LABELS"]
+        elif engine == 'tigre':
+            if isinstance(geometry, BaseAcquisitionGeometry):
+                dim_order = data_order["TIGRE_AG_LABELS"]
+            else:
+                dim_order = data_order["TIGRE_IG_LABELS"]
+        elif engine == 'cil':
+            if isinstance(geometry, BaseAcquisitionGeometry):
+                dim_order = data_order["CIL_AG_LABELS"]
+            else:
+                dim_order = data_order["CIL_IG_LABELS"]
+        else:
+            raise ValueError("Unknown engine expected one of {0} got {1}".format(data_order["ENGINES"], engine))
+
+        dimensions = []
+        for label in dim_order:
+            if label in geometry.dimension_labels:
+                dimensions.append(label)
+
+        return dimensions
+
+image_labels: ImageLabels = {"RANDOM": "random",
+                             "RANDOM_INT": "random_int",
+                             "CHANNEL": "channel",
+                             "VERTICAL": "vertical",
+                             "HORIZONTAL_X": "horizontal_x",
+                             "HORIZONTAL_Y": "horizontal_y"}
+
+acquisition_labels: AcquisitionLabels = {"RANDOM": "random",
+                                         "RANDOM_INT": "random_int",
+                                         "ANGLE_UNIT": "angle_unit",
+                                         "DEGREE": "degree",
+                                         "RADIAN": "radian",
+                                         "CHANNEL": "channel",
+                                         "ANGLE": "angle",
+                                         "VERTICAL": "vertical",
+                                         "HORIZONTAL": "horizontal",
+                                         "PARALLEL": "parallel",
+                                         "CONE": "cone",
+                                         "DIM2": "2D",
+                                         "DIM3": "3D"}
+
+data_order: DataOrder = \
```

Comment:
The documentation for DataOrder should either be expanded on or removed.
Expanded on could be useful for developers, but argument to remove is that users wouldn't use this class
![image](https://github.com/user-attachments/assets/ef6821e3-49e1-49c5-96cb-59ea046b173a)

Reply:
I removed it but added in the labels instead.